crates/protos/src/bussiness.proto

```diff
@@ -21,9 +21,10 @@ message HPoints {
 // ZMessage.type = Z_TYPE_GATEWAY
 // Gateway just only needs read api
 message ZGateway {
-    GatewayType type = 1;
-    QueryMethod method = 2;
-    bytes data = 3;
+    string request_id = 1;
```

Comment:
You don't need to add an ID field here, it should be handled at the RPC level, just like eth rpc